src/addElasticsearchNode.ts

```diff
@@ -0,0 +1,6 @@
+export async function handler(event) {
+    return new Promise((resolve, reject) => {
+        resolve("done");
```

Comment:
What are the benefits of resolving with `"done"`? Could we resolve `true`? 🤔 

(no strong opinion - just thinking out loud)

Reply:
Ah - nevermind me, I see it gets [used here](https://github.com/guardian/elasticsearch-node-rotation/pull/7/files#diff-cb222d20613245c796d9ce29c4fb26a8R127)

Reply:
There is no real reason for this, I picked at random (since I just needed a placeholder lambda to point at in the Step Function definition)